examples/custom-pod-network-cidr/README.md

```diff
@@ -0,0 +1,37 @@
+# Custom pod network cidr
+
+If the cluster has another pod network cidr different than `172.16.0.0/16` you can create a `patch.yaml` file:
+
+```yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: calico-node
+spec:
+  template:
+    spec:
+      containers:
+        - name: calico-node
+          env:
+          - name: CALICO_IPV4POOL_CIDR
+            value: "172.16.0.0/16" # PUT YOUR NETWORK CIDR HERE
```

Comment:
Can we put another CIDR in the example? Just not to confuse users.

Reply:
sure!

Reply:
You've marked this as resolved, but I think you forgot to push the changes or maybe you changed only the patch and forgot to also change the README?

Reply:
My fault. Updated ;) Thanks!